api/cmd/api/setup.go

```diff
@@ -506,17 +463,11 @@ func initVersionEndpointService(cfg *config.Config, builder imagebuilder.ImageBu
 	})
 }
 
-func initLogService(cfg *config.Config, vaultClient vault.Client) service.LogService {
-	// image builder cluster
-	imgBuilderClusterSecret, err := vaultClient.GetClusterSecret(cfg.ImageBuilderConfig.ClusterName)
-	if err != nil {
-		log.Panicf("unable to retrieve secret for cluster %s from vault: %v", cfg.ImageBuilderConfig.ClusterName, err)
-	}
+func initLogService(cfg *config.Config) service.LogService {
+	creds := mlpcluster.NewK8sClusterCreds(&cfg.ImageBuilderConfig.K8sConfig)
 	ctl, err := cluster.NewController(cluster.Config{
-		Host:       imgBuilderClusterSecret.Endpoint,
-		CACert:     imgBuilderClusterSecret.CaCert,
-		ClientCert: imgBuilderClusterSecret.ClientCert,
-		ClientKey:  imgBuilderClusterSecret.ClientKey,
+		Host:        cfg.ImageBuilderConfig.K8sConfig.Cluster.Server,
```

Comment:
Host and other fields should be cleaned up now

Reply:
Done

Reply:
Removed.